package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "notifications-panel",
-  "version": "1.2.1",
+  "version": "1.3.0",
```

Comment:
it changes the api. 

hypothetically if a client were expecting OPEN_LINKs they will be unpleasantly surprised that now they are getting OPEN_SITE